README.md

```diff
@@ -1,4 +1,11 @@
-[![Puppet Forge](http://img.shields.io/puppetforge/v/puppet/letsencrypt.svg?style=flat)](https://forge.puppetlabs.com/puppet/letsencrypt) [![Build Status](https://travis-ci.org/voxpupuli/puppet-letsencrypt.svg)](https://travis-ci.org/voxpupuli/puppet-letsencrypt) [![Documentation Status](http://img.shields.io/badge/docs-puppet--strings-ff69b4.svg?style=flat)](http://voxpupuli.github.io/puppet-letsencrypt)
+# Let's Encrypt
+
+[![Build Status](https://travis-ci.org/voxpupuli/puppet-letsencrypt.png?branch=master)](https://travis-ci.org/voxpupuli/puppet-letsencrypt)
+[![Code Coverage](https://coveralls.io/repos/github/voxpupuli/puppet-letsencrypt/badge.svg?branch=master)](https://coveralls.io/github/voxpupuli/puppet-letsencrypt)
```

Comment:
There is no `lib` directory, so coveralls is never run.  For modules like this, we shouldn't include a coveralls badge?